src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredWoman = century === undefined
+    ? people.filter(man => man.sex === 'm')
+    : people.filter(man => man.sex === 'm'
+    && Math.ceil(man.died / 100) === century);
```

Comment:
* There are a lot of magic strings and numbers here, consider using variables with meaningful names. 
* There is repeating filtering functionality, consider using isMan() or isWoman function to figure out the sex of the person